packages/osc-recruitment/app/queries/fragments/buildUrls.ts

```diff
@@ -0,0 +1,14 @@
+import groq from 'groq';
+import { PATHS } from '~/constants';
+
+export const buildUrls = groq`
+    (_type == "home") => {
+        "slug": "${PATHS.HOME}",
+    },
+    (_type == "page") => {
+        "slug": '/' + slug.current,
+    },
+    (_type == "job") => {
+        "slug": "/${PATHS.JOBS}/" + store.slug.current,
```

Comment:
Can you take `store.` off this? so it should just be `slug.current` like this
```suggestion
        "slug": "/${PATHS.JOBS}/" + slug.current,
```